e2e/pages/events/actionCaseManagementOrderEvent.page.js

```diff
@@ -79,4 +79,9 @@ module.exports = {
     I.waitForElement(this.fields.nextHearingDateList);
     I.selectOption(this.fields.nextHearingDateList, date);
   },
+
+  async enterDateOfIssue(date){
+    I.fillDate(date);
+    await I.retryUntilExists(() => I.click('Continue'), '#allPartiesCustomCMO');
```

Comment:
could move #allPartiesCustomCMO into the fields section of this file, up to you if you'd prefer leave as is - very minor